wiki/assets/temp.txt

```diff
@@ -0,0 +1 @@
+I'm just here so I dont get fined.
```

Comment:
Uh...

Reply:
Oh, I just needed a placeholder file to create the folder...cause I was doing all this work on the web, I just forgot to go back and delete it haha

Reply:
No worries, but if you're interested, the convention is to create an empty file called `.gitkeep`